.gitignore

```diff
@@ -84,6 +84,8 @@ venv/
 .installed.cfg
 *.egg
 
+!torchci/lib
```

Comment:
why was this needed?

Reply:
`lib/` is in the gitignore for distribution and packaging, I'm not really sure what it's trying to ignore, so I decided to be cautious and just enable `torchci/lib`